src/app/components/event/steps/Cover.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="flex flex-col gap-6">
+    <div class="flex flex-col gap-4">
+      <h2
+        class="text-lg font-semibold dark:text-[--semantic-base-text-primary]"
+      >
+        {{ selectedFiles.length ? t('selectCover') : t('addImages') }}
+      </h2>
+      <EventFileUploadEmptyState
+        v-if="!selectedFiles.length"
+        @trigger-upload="triggerFileInput"
+      />
+      <template v-else>
+        <EventFilePreviewGrid
+          :files="selectedFiles"
+          :preview-urls="previewUrls"
+          v-model="selectedCover"
```

Comment:
`v-model="selectedCover"` shows a warning because this property should come first after `<EventFilePreviewGrid`